.github/config.yml

```
@@ -0,0 +1,9 @@
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+  🎉🥳 Congratulations on getting your very first Zephyr pull request merged! This is a fantastic achievement, and we're thrilled to have you as part of our community!
+
+  To celebrate this milestone and showcase your contribution, we'd love to award you the Zephyr Technical Contributor badge. If you're interested, please claim your badge by filling out this form: (Claim Your Zephyr Badge)[https://forms.gle/oCw9iAPLhUsHTapc8].
```

Incorrect link format

```suggestion
  To celebrate this milestone and showcase your contribution, we'd love to award you the Zephyr Technical Contributor badge. If you're interested, please claim your badge by filling out this form: [Claim Your Zephyr Badge](https://forms.gle/oCw9iAPLhUsHTapc8).
```